.github/workflows/integration-tests.yml

```diff
@@ -118,9 +123,18 @@ jobs:
             exit 1
           fi
           
-          if grep -r "crashed" "$zkvm_folder"/*.json 2>/dev/null; then
-            echo "Found 'crashed' string in ZKVM folder JSON files"
-            exit 1
+          if [ "${{ matrix.test-type }}" = "invalid-blocks" ]; then
+            # For invalid-blocks, we expect all tests to crash - fail if any success is found
+            if grep -r "success" "$zkvm_folder"/*.json 2>/dev/null; then
+              echo "Found unexpected 'success' string in invalid-blocks test - some tests succeeded when they should have failed"
+              exit 1
+            fi
```

Comment:
To avoid blocking this PR on ere supporting reporting outputs, for now I check that the run crashed (since the guest program today panics if the stateless validation of the block fails).

Whenever ere supports reporting outputs, we can easily check outputs instead of doing this. (sidenote: since I'll be moving integration tests into Rust code, it will also be nicier than looking from strings in json files which I think is messy).